tests/provisioning-manifest-build.yml

```diff
@@ -1,2 +1,10 @@
 - addMavenRepository: 'https://store.jahia.com/nexus/content/repositories/jahia-public-app-store@id=JahiaStore'
 - addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/'
+
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/@snapshots@noreleases'
+- installBundle:
+    - 'mvn:org.jahia.modules/jcontent'
```

Comment:
if this is build provisioning, shouldn't we be using the recently built version, not the one from maven (unless they're the same in this case)?

Reply:
Should not have been committed, and it was wrong